eng/Signing.props

```diff
@@ -1,9 +1,23 @@
 <Project>
+
+  <Import Project="MPack.props" />
+
   <!--
     These are third party libraries that we use in this repo. We need to sign them even if they
     are already signed. However, they must be signed with a 3rd party certificate.
   -->
   <ItemGroup>
     <FileSignInfo Include="Newtonsoft.Json.dll" CertificateName="3PartySHA2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ItemsToSign Include="$(ArtifactsMPackDir)**\*.mpack" />
```

Comment:
Duplicate of the addition to MPack.props. Remove?

Reply:
Ah, right. I ended up putting it in both places.